src/pages/settings/setup.tsx

```diff
@@ -34,8 +35,17 @@ const Setup: NextPage = () => {
             <Grid item lg={4} xs={12}>
               <Typography variant="h4" fontWeight={700}>
                 Safe nonce
+                <Tooltip title="For security reasons, transactions made with Safe need to be executed in order. The nonce shows you which transaction will be executed next. You can find the nonce for a transaction in the transaction details.">
```

Comment:
I'd add `placement="top"` to the `Tooltip` so you can still read the current nonce. Most `Tooltip`s in the app use this.

Reply:
Good idea!